src/client.c

```diff
@@ -67,6 +63,19 @@ int choose_tile_handler(int c, rcb_t *rcb)
         render_board_preview(rcb);
         break;
     }
+    case 'H':
+        shift_msg(msg, color);
+        snprintf(msg[6], MAX_LOG_LEN,
+                 "[Hint] You could place tile \"%c\" at (%x, %x)",
+                 TILE[gcb->hint.shape].alpha, gcb->hint.pos.y, gcb->hint.pos.x);
+        *color[6] = BLUE_PAIR;
+        render_message_log(msg, color);
+    default:
+        shift_msg(msg, color);
+        snprintf(msg[6], MAX_LOG_LEN, "[Warning] Invalid operation \"%c\"", c);
```

Comment:
Directly print `c` may break UI when pressing enter.
![image](https://user-images.githubusercontent.com/79467307/170911908-cc1fc1bd-8f14-4b5b-8f24-ad09f12c6f8d.png)

Reply:
Some other non-printable characters such as control characters or combinations of ctrl + some keys, for example, may also break UI.